cmd/tools/genrpcwrappers/main.go

```diff
@@ -617,11 +591,11 @@ func main() {
 
 	i := slices.IndexFunc(services, func(s service) bool { return s.name == *serviceFlag })
 	if i < 0 {
-		panic("unknown service")
+		codegen.FatalIfErr(fmt.Errorf("unknown service: %s", *serviceFlag))
```

Comment:
I might like to see `codegen.Fatal("unknown service: %s", *serviceFlag)` since this pattern is all over the place?